cinch/library/jenkins_plugin.py

```diff
@@ -0,0 +1,826 @@
+#!/usr/bin/python
```

Comment:
I meant a git submodule.  I take it you did perform some modification of the original code?  If that's the case then we can leave it as-is.  I thought it was brought in wholesale from another repo.